style.css

```diff
@@ -0,0 +1,468 @@
+.body{
+       margin:0px;
+       /* background-color:black */
+}
+.header{
+    clear:both;
+    z-index: 1;
+    max-width: 1830px;
+    margin-top: 54px;
+}
+
+.header__container{
+    margin:0px 344px 0px 360px; 
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+}
+
+.logo__image{
+    /* background-color:black; */
+    max-width: 78px;
+    max-height: 64px;
+}
+
+.navbar {
+    display:inline-block;
+    max-width: 504px;
+    max-height: 21px;
+}
+
+
+.nav__items{
+    display:flex;
+    max-width: 504px;
+    max-height: 21px;
+    list-style:none;
+}
+
+.nav__items:hover::after{

```

Comment:
Vizual: bulinuta nu este unde trebuie, si e pus hover pe items, ar trebui defapt sa fie pe fiecare item in parte